experimental/sql/src/impl_to_iterators.py

```diff
@@ -258,6 +258,21 @@ def match_and_rewrite(self, op: RelImpl.FullTableScanOp,
                                  convert_bag(op.result.typ)))
 
 
+@dataclass
+class PartialTableScanRewriter(RelImplRewriter):
```

Comment:
Only realizing this now: Is there any reason not to make "partial" scans an option of the full scan in this dialect as well?

Reply:
This comment is out of date, right?